src/Command/TranskribusAuthCommand.php

```diff
@@ -0,0 +1,86 @@
+<?php declare(strict_types = 1);
+
+namespace App\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\Question;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+class TranskribusAuthCommand extends Command
+{
+
+    /** @var HttpClientInterface The interface to make http requests */
+    private $client;
+
+    /** @var string The client ID passed to the transkribus API */
+    private $clientId;
+
+    public function __construct(HttpClientInterface $client, string $clientId)
+    {
+        parent::__construct();
+        $this->client = $client;
+        $this->clientId = $clientId;
+    }
+
+    protected function configure(): void
+    {
+        $this
+            ->setName('app:transkribus')
+            ->setDescription('This command allows you to retrieve and store an access token for the Trankribus API');
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $helper = $this->getHelper('question');
+        $io = new SymfonyStyle($input, $output);
+
+        $userNameQuestion = new Question('Enter username: ');
```

Comment:
Before asking for username and continuing, this should look for `APP_TRANSKRIBUS_ACCESS_TOKEN` and if it's found it should use that to check and report the current authentication status.

Reply:
I'm not sure about how to check the current authentication status using `APP_TRANSKRIBUS_ACCESS_TOKEN`. I'll take a look at it and try including that in the next commit